CMakeLists.txt

```diff
@@ -0,0 +1,228 @@
+cmake_minimum_required(VERSION 3.1)
+
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/cmake/toolchain.cmake"
+    CACHE FILEPATH
+    "Toolchain to use for building this package and dependencies")
+
+project(pcg VERSION 1.0.0)
```

Comment:
> I was going with 1.0.0 because it hasn't changed in a while

It's better to set `0.0.0` to avoid conflicts with future versions.

Reply:
Got it.